### PR TITLE
Add target to run tests as binary

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- run tests as binary

**Related issue(s)**
https://github.com/kyma-project/serverless-manager/issues/44

### CODE REVIEW

File: tests/function-controller/run-local.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+kubectl proxy &
+printenv  | grep DAMIAN
```
Comment (anoipm) [inline tests/function-controller/run-local.sh:3]: `grep DAMIAN` ???

Comment (dbadura) [reply tests/function-controller/run-local.sh]: Fixed :)

**Review by anoipm — APPROVED** [review]

**Pull request merged**